pkg/verify/verify.go

```diff
@@ -43,6 +44,19 @@ type Interface interface {
 	AddStore(additionalStore store.Store)
 }
 
+type wrapError struct {
+	msg string
+	err error
+}
+
+func (e *wrapError) Error() string {
```

Comment:
`wrapError` is [from Go][1], because I wanted `fmt.Sprintf("...%w")`'s `Unwrap` support, but without formatting the whole, possibly long aggregate error into the message.  We need to use `Error` and `Unwrap` to match the `error` interface.

[1]: https://cs.opensource.google/go/go/+/refs/tags/go1.18.1:src/fmt/errors.go;l=32-43;drc=refs%2Ftags%2Fgo1.18.1